### PR TITLE
Feature/my past session

### PR DESCRIPTION
**Pull request by Nituyadav0109**

### CODE REVIEW

File: go1percent/test/Go1PercentAPIAutomation/myPastSession/myPastSession.js
Comment (Ritika-gupta20) [inline go1percent/test/Go1PercentAPIAutomation/myPastSession/myPastSession.js:1]: Folder name to be in camel case

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Please resolve the comments

File: go1percent/test/Go1PercentAPIAutomation/myPastSession/myPastSession.js
Comment (Nituyadav0109) [reply go1percent/test/Go1PercentAPIAutomation/myPastSession/myPastSession.js:1]: renamed the file in  camel case

File: go1percent/package.json
Comment (Ritika-gupta20) [inline go1percent/package.json:1]: please do not push changes in package.json on git

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Please copy the package.json from main and paste it to your branch and push it so that no changes are there

File: go1percent/package.json
Comment (Nituyadav0109) [reply go1percent/package.json:1]: okay

**Review by Ritika-gupta20 — APPROVED** [review]

**Review by deepansh257 — APPROVED** [review]

**Pull request merged**